config.yaml

```diff
@@ -0,0 +1,22 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+options:
+  external_hostname:
+    description: |
+      The external hostname to use. Will default to the name of the deployed
+      application.
+    default: ""
+    type: string
+  github_webhook_token:
+    type: string
+    description: GitHub Webhook Token
```

Comment:
Should this description be expanded? At the moment it just seems to re-state the name of the option

Reply:
Sure, rephrased, thanks.